### ISSUE TITLE
Highlighting text in the Qubes OS Update tool results in black text on a dark blue background, which is difficult to read

### ISSUE THREAD

**Issue by emanruse** [QubesOS/qubes-issues#8949]
### Qubes OS release

4.2

### Brief summary

Black text on dark blue background is difficult to read. Use readable color combination.

### Steps to reproduce

Run Qubes OS Update and select any VM from the list.

### Expected behavior

The selected item should be readable.

### Actual behavior

The text (black on blue) is difficult to read.

**Comment by alimirjamali** [comment]
Just a suggestion here. [Qubes Blue](https://www.qubes-os.org/doc/visual-style-guide/) looks very nice on light themes

css selector is `treeview:selected {background-color: #63a0ff}` for light theme.

![updater](https://github.com/user-attachments/assets/816b1d5a-1567-4a2f-b802-58529c4f0e8f)

**Comment by AnushkaShreyam** [comment]
Hi @marmarta,

I have a possible solution for the text visibility issue in the Qubes OS Update tool. By fixing the background color and changing the text color to ensure better contrast, we can make the UI more readable regardless of the theme the user is using.

For example, using a light text color (like white) against the current dark blue background would enhance readability significantly. This change should help improve the user experience across all themes. 

Similar to what @alimirjamali said we can add: 

treeview:selected {
    background-color: #63a0ff;
    color: #f5f5f5; /* Change text color to gray */
}

Let me know your thoughts! 

Or maybe write the function for adaptive coloring based on user's theme?

**Comment by marmarek** [comment]
Can you post a screenshot how it looks?

**Comment by marmarta** [comment]
seconded the question for a screenshot - and also, I think that generlly this can be a sufficient solution, as the background color does not adapt to theme (I think)
